packages/widgets/src/dns-hole/controls/TimerModal.tsx

```diff
@@ -8,11 +8,11 @@ import { useI18n } from "@homarr/translation/client";
 interface TimerModalProps {
   opened: boolean;
   close: () => void;
-  integrationIds: string[];
+  selectedIntegration: string[];
```

Comment:
This is the singular form. However, this is an array (hence pluaral should be used).
Also, ``integration`` sounds like an object. Please name it ``integrationIds`` since it passes the IDs